internal/builtins/pkg_context.go

```diff
@@ -131,3 +131,18 @@ data: {}
 	})
 	return cm, nil
 }
+
+// DummyPkgContext returns content for package context that contains
+// placeholder value for package name. This will be used to create
+// abstract blueprints.
+func DummyPkgContext() string {
+	return fmt.Sprintf(`apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: %s
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  name: example
+`, pkgContextName)
```

Comment:
Can we use this `DummyPkgContext` to replace lines 106-114 above, since the name is being set in lines 129-130 anyways? Then we will not have to write out this yaml twice.

Reply:
That's good point.  Done!